Extension/src/background/services/locale-detect.ts

```diff
@@ -55,18 +57,45 @@ export class LocaleDetect {
         // German
         'de': 'de',
         'at': 'de',
+        'li': 'de',
         // Japanese
         'jp': 'ja',
         // Dutch
         'nl': 'nl',
+        'sr': 'nl',
         // French
         'fr': 'fr',
+        'mc': 'fr',
+        'ht': 'fr',
         // Spanish
         'es': 'es',
+        'mx': 'es',
+        'ar': 'es',
+        'cl': 'es',
+        'uy': 'es',
+        'pe': 'es',
+        've': 'es',
+        'ec': 'es',
+        'bo': 'es',
+        'py': 'es',
+        'pa': 'es',
+        'cr': 'es',
+        'ni': 'es',
+        'hn': 'es',
+        'gt': 'es',
+        'sv': 'es',
+        'do': 'es',
+        'pr': 'es',
+        'cat': 'es',
         // Italian
         'it': 'it',
+        'sm': 'it',
         // Portuguese
         'pt': 'pt',
+        'br': 'pt',
+        'ao': 'pt',
+        'mz': 'pt',
+        'cv': 'cv',
```

Comment:
'pt' locale is missing

Reply:
https://github.com/AdguardTeam/FiltersRegistry/blob/master/tags/metadata.json#L70-L73 claims the locale exists, if I've looked at the correct place.

Reply:
Should be correct now.